scripts/mbedtls_framework/c_parsing_helper.py

```diff
@@ -129,3 +151,16 @@ def read_function_declarations(functions: Dict[str, FunctionInfo],
                                        return_type,
                                        name,
                                        arguments)
+_C_TYPEDEF_DECLARATION_RE = r'typedef (?:struct ){0,1}(?P<type>\w+) (?P<name>\w+)'
```

Comment:
Could we have a blank line to separate this code from the function above? And maybe get rid of some of the many blank lines at line 157?